tests/test_scheduler.py

```diff
@@ -398,26 +426,31 @@ def test_variance(self):
         assert torch.sum(torch.abs(scheduler._get_variance(999, 998) - 0.02)) < 1e-5
 
     def test_full_loop_no_noise(self):
```

Comment:
@patri in https://github.com/huggingface/diffusers/commit/394243ce98c79e273732f8e09b2e5c50ee9a9bd7 you change PNDM step to accept the timestep value rather than iteration number, but I think the tests were kept as is. I changed the full loop to use actual timesteps.